BiconeEvolution/current_antenna_evo_build/XF_Loop/Evolutionary_Loop/Antenna_Performance_Metric/polar_plotter_v2.py

```diff
@@ -78,9 +83,14 @@
 
 ### Saves these indices to variables
 ### Each individual has 2 files, so the index is multiplied by 2 and subtracted by 1
-max_index = (int(f1.readline())*2)-1
-mid_index = (int(f2.readline())*2)-1
-min_index = (int(f3.readline())*2)-1
+if g.SYMMETRY == 0:
+	max_index = (int(f1.readline())*2)-1
+	mid_index = (int(f2.readline())*2)-1
+	min_index = (int(f3.readline())*2)-1
+else:  
+	max_index = (int(f1.readline()))-1
+	mid_index = (int(f2.readline()))-1
+	min_index = (int(f3.readline()))-1
```

Comment:
```suggestion
if g.SYMMETRY == 0:
	max_index = int(f1.readline())*2 - 1
	mid_index = int(f2.readline())*2 - 1
	min_index = int(f3.readline())*2 - 1
else:  
	max_index = int(f1.readline()) - 1
	mid_index = int(f2.readline()) - 1
	min_index = int(f3.readline()) - 1
```
To avoid duplication you could also set a multiplier to `2` if the symmetry is `0`, or `1` if it's not, and then just use `int(...) * mul - 1`.  The same could be done above.

Reply:
Ok, I added a variable to multiply by for both areas where it's used.